jasper_server/data/jasper_document_extension_data.xml

```diff
@@ -24,65 +24,65 @@
         ##############################################################################
 
 
+        <!-- TODO: The following records are commented because currently they are not working (v10.0.1.0)
         <record model="jasper.document.extension" id="jasper_document_extension_1">
-            <field name="jasper_code">ODS</field>
             <field name="name">OpenOffice Writer</field>
-            <field name="extension">ods</field>
+            <field name="extension">odt</field>
         </record>
+        -->
 
         <record model="jasper.document.extension" id="jasper_document_extension_2">
-            <field name="jasper_code">PDF</field>
```

Comment:
It was redundant, if you take a look at the extension and jasper_code, they are similar but the jasper_code is on upper_case.
Moreover, this field was being used only to set the values of the Selection field from a jasper.document, but now I changed the selection for a many2one (which I think is more correct), and it was not used anymore.
I think is better to delete it to avoid duplicity and a source of errors